lib/components/Menu.js

```diff
@@ -120,16 +120,18 @@ var Menu = module.exports = React.createClass({
   renderMenuOptions: function() {
     var options;
     if(this.verifyTwoChildren()) {
-      React.Children.forEach(this.props.children, function(child){
-        if (child.type === MenuOptions.type) {
-          options = cloneWithProps(child, {
-            ref: 'options',
-            horizontalPlacement: this.state.horizontalPlacement,
-            verticalPlacement: this.state.verticalPlacement,
-            onSelectionMade: this.closeMenu
-          });
-        }
-      }.bind(this));
+      if (this.state.active) {
```

Comment:
I'd probably refactor this to just put this at the top:
```
if(!this.state.active) { return null; }
```

Because react will interpret the `null` as placeholder which will help it render the list when active.